testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -190,6 +191,10 @@ def universe2(self):
     def universe3(self):
         return mda.Universe(PDB)
 
+    @pytest.fixture
+    def universe4(self):
+        return mda.fetch_mmtf("2BBM")
```

Comment:
Bundle the file as bzip2-compressed file or even better, just a minimal example.

We should not make our test suite depend on internet connectivity. Also, the file can be changed externally and that will compromise our tests.

Reply:
Very good points, thanks!

Reply:
If I download the `2BBM` PDB file and use that for testing, the test fails because the unit cell is set to `[0.0, 0.0, 0.0, 0.0, 0.0, 0.0]`. @IAlibay Shouldn't `u.dimensions` be `None` after #2645? If I missed something, I can add a check for `[0.0, 0.0, 0.0, 0.0, 0.0, 0.0]` in my fix.

Reply:
#2645 only attempted to handle cryo-em default cases, if there are more default cases than [1,1,1,90,90,90] then those will need handling.

Reply:
It dimensions is `None` with `fetch_mmtf("2BBM")` but `[0.0, 0.0, 0.0, 0.0, 0.0, 0.0]` if I load `2bbm.pdb.gz` manually.

Reply:
Yeah, so the MMTFReader specifically sets `_unitcell` to `None`, whilst we otherwise just default to whatever is set to ReaderBase in the PDBReader (as mentioned below, this would probably need alignment in behaviour).